examples/servicegraph/cmd/server/BUILD

```diff
@@ -1,4 +1,4 @@
-package(default_visibility = ["//visibility:private"])
+package(default_visibility = ["//visibility:public"])
```

Comment:
Is there any reason to make this public?  I don't think we want anything outside of examples/servicegraph to be importing the library here.  Let's keep a small scope, if possible.